src/components/accordion/components.test-pw.tsx

```diff
@@ -7,28 +7,6 @@ import { Checkbox } from "../checkbox";
 import { Dl, Dt, Dd } from "../definition-list";
 import Link from "../link/link.component";
 
-export const Grouped = () => (
```

Comment:
**question(non-blocking):** was this meant to be deleted? We have a Chromatic snapshot capturing this test story that is all.

Reply:
I deleted this as it seemed copied over from the test stories and was not being used anywhere, I also don't think we have any snapshots for this component as its set to `disableSnapshot: true` on the test stories.